### PR TITLE
Hergoku

### PR DESCRIPTION
**Pull request by vishalkuo**
- This guy isn't ready yet but I'd like to consolidate these branches once sqlite is sucessfully working

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  rtm := api.NewRTM()
+  go spawn_server()
+  go initiate_get_request()
+  go rtm.ManageConnection()
+  for {
+    select {
+    case msg:= <-rtm.IncomingEvents:
+      switch ev := msg.Data.(type) {
+        case *slack.MessageEvent:
+          go messageHandler(ev, rtm)
```
Comment (tanatorn) [inline main.go:46]: Maybe buffer this - probably not gonna be a problem right now, but make it so it's like only some threshold event at the same time?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    key = getKey(logger)
+  }
+  api := slack.New(key)
+  slack.SetLogger(logger)
+  logger.Print("Initiated slack client")
+  return api
+}
+
+func getKey(logger *log.Logger) string {
+  data, err := ioutil.ReadFile("keys/api_key.txt")
```
Comment (tanatorn) [inline main.go]: if we're using Heroku, can't we just use `heroku local` and put the keys into an `.env` file?

File: main.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    resp, err = http.Get("http://uw-ezebot.herokuapp.com")
+  }
+  if err != nil {
+    panic(err)
+  }
+  defer resp.Body.Close()
+  time.Sleep(20 * time.Second)
+}
+
+func spawn_server(){
```
Comment (tanatorn) [inline main.go]: isn't go camel case?

**Review by tanatorn — CHANGES_REQUESTED** [review]
Ay

File: main.go (hunk 2 above)
Comment (vishalkuo) [reply main.go]: Good idea, I can't commit the slack key publicly (slack actually disables the key if it finds it on the web) so... Just message me for the env file

Comment (vishalkuo) [reply main.go]: Actually I can't even get it to work at the moment. Let me know if you have better luck

**Review by tanatorn — APPROVED** [review]

**Comment by tanatorn** [comment]
LGTM

**Pull request merged**